src/os/bluestore/BlueStore.cc

```diff
@@ -4515,6 +4515,7 @@ BlueStore::BlueStore(CephContext *cct,
 #endif
     min_alloc_size(_min_alloc_size),
     min_alloc_size_order(ctz(_min_alloc_size)),
+    m_bluestore_fail_eio(cct->_conf->bluestore_fail_eio),
```

Comment:
Please add the initialization to other ctor(s)

Reply:
could you please explain a little further, or where should it supposed to be. thanks

Reply:
There is a different BlueStore's ctor:
BlueStore::BlueStore(CephContext *cct, const string& path)
  : BlueStore(cct, path, 0) {}

which lacks the new member initialization.

after another glance IMO one might avoid that intermediate class member usage at all and access cct->_conf->bluestore_fail_eio directly wherever needed, e.g.
IOContext ioc(cct, NULL, !cct->_conf->bluestore_fail_eio);

Reply:
Modifications have been made based on the feedback, thanks